src/components/OpportunityCard/OpportunityCard.tsx

```diff
@@ -211,13 +213,20 @@ export const OpportunityCard = ({
   return (
     <>
       {loginModalOpen && <LoginModal onClose={() => setLoginModalOpen(false)} isOpen={loginModalOpen} />}
-      <div className="rounded shadow-border bg-white pt-5 pr-5 pl-6 pb-7">
-        <div className="flex flex-col gap-3 sm:flex-row sm:gap-5">
+      <div className="rounded shadow-border bg-white pt-5 pr-5 pl-5 sm:pl-6 pb-5 sm:pb-7">
+        <div className="flex flex-col sm:flex-row sm:gap-5">
           <div className="flex flex-row justify-between align-center">
-            {matchValue && matchLabel && <Match match={matchValue} label={matchLabel} bg={bgForType(type)} />}
+            <div>
```

Comment:
Added a `div` tag, so that if the `<Match />` is not shown, the space is still reserved and favorite and checkbox stays in the right side of the card.